std/experimental/typecons.d

```diff
@@ -802,3 +802,223 @@ private template findCovariantFunction(alias finfo, Source, Fs...)
     else
         enum ptrdiff_t findCovariantFunction = x;
 }
+
+/++
```

Comment:
Nitpick: `/+ +/` comments should be used when needed (e.g. when nesting is required), for ddoc stuff we use `/** */` so we should keep it consistent.